src/search/components/SearchResults.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+
+const propTypes = {
+  query: PropTypes.string,
+  results: PropTypes.string,
+};
+const defaultProps = {
+  query: '',
+  results: [],
+};
+
+class SearchResults extends React.Component {
+  render() {
+    const { query, results } = this.props;
+    return (
+      <div className="search-results">
+        <h3>Your search for "{query}" returned {results.length} results.</h3>
+        {results.map((result) => <div key={result.title}>{result.title}</div>)}
+      </div>
+    );
+  }
+}
+
+SearchResults.propTypes = propTypes;
+SearchResults.defaultProps = defaultProps;
```

Comment:
This would be another reason to use class props from stage 1. Using class props for this is quite a bit nicer imho. Also after all rest spread operator is still only a draft (stage 2).

I'd be surprised if class props didn't make it into the spec. Absolute worst case - we'd have to change them back.